README.md

```diff
@@ -16,122 +16,146 @@ NumPyro is a small probabilistic programming library that provides a NumPy backe
 
 ```python
+>>> from numpyro.infer.reparam import TransformReparam
+
 >>> # Eight Schools example - Non-centered Reparametrization
 ... def eight_schools_noncentered(J, sigma, y=None):
 ...     mu = numpyro.sample('mu', dist.Normal(0, 5))
 ...     tau = numpyro.sample('tau', dist.HalfCauchy(5))
 ...     with numpyro.plate('J', J):
-...         theta = numpyro.sample('theta', 
-...                                dist.TransformedDistribution(dist.Normal(0., 1.),
-...                                                             dist.transforms.AffineTransform(mu, tau)))
+...         with numpyro.handlers.reparam(config={'theta': TransformReparam()}):
+...             theta = numpyro.sample(
+...                 'theta',
+...                 dist.TransformedDistribution(dist.Normal(0., 1.),
+...                                              dist.transforms.AffineTransform(mu, tau)))
 ...         numpyro.sample('obs', dist.Normal(theta, sigma), obs=y)
 
 >>> nuts_kernel = NUTS(eight_schools_noncentered)
 >>> mcmc = MCMC(nuts_kernel, num_warmup=500, num_samples=1000)
 >>> rng_key = random.PRNGKey(0)
 >>> mcmc.run(rng_key, J, sigma, y=y, extra_fields=('potential_energy',))
->>> mcmc.print_summary()
-
-                mean       std    median      5.0%     95.0%     n_eff     r_hat
-        mu      4.38      3.04      4.50     -0.92      9.05    876.02      1.00
-       tau      3.36      2.89      2.63      0.01      7.56    755.65      1.00
-  theta[0]      5.99      5.42      5.44     -1.33     15.13    825.18      1.00
-  theta[1]      4.80      4.50      4.78     -1.63     13.01   1114.97      1.00
-  theta[2]      3.94      4.63      4.23     -3.41     11.06    914.68      1.00
-  theta[3]      4.76      4.62      4.73     -2.31     12.11    958.40      1.00
-  theta[4]      3.62      4.66      3.75     -3.87     11.17   1091.53      1.00
-  theta[5]      3.92      4.43      4.06     -2.41     11.09   1179.74      1.00
-  theta[6]      5.88      4.84      5.34     -1.45     13.11    881.38      1.00
-  theta[7]      4.63      4.86      4.64     -3.57     11.80   1065.27      1.00
-
+>>> mcmc.print_summary(exclude_deterministic=False)
+<BLANKLINE>
+                   mean       std    median      5.0%     95.0%     n_eff     r_hat
+           mu      4.08      3.51      4.14     -1.69      9.71    720.43      1.00
+          tau      3.96      3.31      3.09      0.01      8.34    488.63      1.00
```

Comment:
`n_eff` seems to have gone down for both `mu` and `tau`. Is this just noise?

Reply:
I think so.